<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddUser</class>
 <widget class="QWidget" name="FormAddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellido">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCedula">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbNombre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>13</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbApellido">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCedula">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cedula:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbProf">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Profesor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbEstu">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estudiante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
